Shadan/style.css

```diff
@@ -0,0 +1,159 @@
+*
+{
+    margin:0;
+    padding:0;
+}
+html{
+    background-color: #f7f7f7;
+}
+header{
+    background-color: #e6e6e6;
+
+}
+
+header .topcenter{
+    position:relative;
+}
+
+nav .logo{
+
+    width:50px;
+    height:50px;
+   position: absolute;
+   left:25%;
```

Comment:
I don't think that its good idea to use `%` with the positioning properties try to change it into `px`